cfy_manager/components/prometheus/prometheus.py

```diff
@@ -109,7 +109,7 @@ class Prometheus(BaseComponent):
 
     def configure(self):
         logger.notice('Configuring Prometheus Service...')
-        _set_selinux_permissive()
+        _set_selinux_enforcing()
```

Comment:
This isn't something that should happen in more than one place even if it is going to happen- it should be part of either install or configure top level command (or preferably, as noted above, not present at all).